Simonyi/Non-Standard-Operations/MTCS/MTPtg/MTPtg-Configuration-for-MTRot-MTMount.rst

```diff
@@ -0,0 +1,194 @@
+
+When either the ``MTRotator`` or ``MTMount`` components become unavailable during Simonyi operations, and you still want to continue testing and tracking,
+it's necessary to change the configuration of the ``MTPtg`` CSC to avoid entering into ``FAULT`` mode commanding these unavailable components.
+
+
+.. _MTMTPtg-Configuration-for-MTRotator-and-MTMount-Procedure-Error-Diagnosis:
+Error Diagnosis
+===============
+
+Cases
+-----
+
+1. ``MTRotator`` is not available, but you still want to track **without** the Rotator using the rest of the components; or you want to **include** 
+the Rotator in the tracking again.
+
+2. ``MTMount`` is not available (not starting up, for example), but you still want to use *CCW* + *Rotator*  **without** moving or commanding the mount, 
+or you want to revert the change and **include** the mount.
+
+
+.. _MTMTPtg-Configuration-for-MTRotator-and-MTMount-Procedure-Procedure-Steps:
+Procedure Steps
+===============
+
+.. warning:: 
+    
+    Announce through the Slack channel *#summit-simonyi* that the component is not available, and you are about to change the configuration.
+    
+    The Commissioning Scientists on shift will change the version of ``MTMount`` CSC in *ArgoCD*; this can deal with tracking without said component.
+
+        The Commissioning Scientists  may request to send the ``MTMount`` to **OFFLINE**. this will change the ``MTMount`` CSC version to **mtmount-ccw-only**. 
+..
+
+
+
+Steps
+-----
+
+1. Issue the :file:`set_summary_state.py` script in LOVE to change the status of ``MTPtg`` to ``STANDBY`` with the following configuration
+
+    .. code-block::
+        :caption: set_summary_state.py
+
+             data:
+                 -
+                   - MTPtg 
+                   - STANDBY
+..
+
+
+2. Find the name of the **pod** where the ``MTPtg`` is running. 
+   
+    From your terminal, run the following command which list all the **pods** related to the *<namespace>* maintel:
+
+    .. prompt:: bash
+
+     kubectl --kubeconfig=${HOME}/.kube/yagan.yaml get pod -o=custom-columns=NAME:.metadata.name,STATUS:.status.phase,NODE:.spec.nodeName -n maintel
+
+    ..
+
+    .. figure:: ./_static/1.png
+      :width: 950px
+      :height: 165px
+      :name: Your figure
+
+      In this particular case the name of the ``MTPtg`` **pod** is **mtptg-djhpv**.
+    ..  
+
+3. Connect to the ``MTPtg`` **pod mtptg-djhjv** within the *<namespace>* maintel. 
+
+    The command will open a terminal within the pod.
+
+    .. prompt:: bash
+
+     kubectl --kubeconfig=${HOME}/.kube/yagan.yaml exec --stdin --tty mtptg-djhpv -n maintel -- /bin/bash
+
+    ..
+   
+    .. figure:: ./_static/2.png
+        :width: 900px
+        :height: 65px
+
+    ..
+
+4. **Move to configuration directory,** the configuration files are one level up. The directory contains the configuration files :file:`MTPtg.info` (:file:`ATPtg.info` for AuxTel) and the pointing models :file:`mt.mod` files (:file:`at.mod` for AuxTel). 
```

Comment:
I would rewrite this, an example: 'Move to the directory that contains the configuration files’
Or ‘Move to the configuration directory indicated below.’'